apps/web/lib/zod/schemas/links.ts

```diff
@@ -611,6 +612,14 @@ export const getLinkInfoQuerySchema = domainKeySchema.partial().merge(
   }),
 );
 
+export const LinkErrorSchema = z.object({
+  link: createLinkBodySchema
+    .partial()
+    .describe("The link that caused the error."),
```

Comment:
This is super helpful. Thanks for adding this!

Reply:
@joeauyeung thank you for catching the issue – appreciate the feedback here!